.gitignore

```diff
@@ -82,4 +82,6 @@ dist/
 /conduit.yaml
 
 # Escape Analysis Report
-escape_analysis.txt
\ No newline at end of file
+escape_analysis.txt
+
+pkg/plugin/processor/standalone/test/wasm_processors/*/processor.wasm
```

Comment:
Yeah, same here, I was surprised to see the size. I initially committed the files, because it adds a bit of time to tests. But when I reviewed the PR myself, and saw the 10 MB... Quite a surprise (TinyGo produces much smaller binaries).